packages/fast-usdc/src/exos/settler.js

```diff
@@ -221,7 +222,7 @@ export const prepareSettler = (
 
             case PendingTxStatus.Advancing:
               log('⚠️ tap: minted while advancing', nfa, amount);
-              this.state.mintedEarly.add(makeMintedEarlyKey(nfa, amount));
+              self.queueMintedEarly(nfa, amount);
```

Comment:
it's not really a queuing operation because there's no distinguishing the items. I think it should still be `add`. But ideally we don't need this separate method because the data store is a multiset that manages repeated items itself.